pyscf/adc/radc_ao2mo.py

```diff
@@ -45,7 +45,7 @@ def transform_integrals_incore(myadc):
     eris.ovvo = ao2mo.general(myadc._scf._eri, (occ, vir, vir, occ), compact=False).reshape(nocc, nvir, nvir, nocc).copy()  # noqa: E501
     eris.ovvv = ao2mo.general(myadc._scf._eri, (occ, vir, vir, vir), compact=True).reshape(nocc, nvir, -1).copy()  # noqa: E501
 
-    if (myadc.method == "adc(2)-x" or myadc.method == "adc(3)"):
+    if (myadc.method == "adc(2)-x" and myadc.higher_excitations == True) or (myadc.method == "adc(3)"):
```

Comment:
Please include "higher excitations" by default everywhere. I suggest to rename the flag to "approx_trans_moments" with a default value of False for the molecular code (equivalent to higher_excitations == True) and default value of True for the PBC code.

Reply:
Done